documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentInfoDrawer.vue

```diff
@@ -261,6 +272,25 @@ export default {
         }
       }));
     },
+    addDescriptionStatistics(file) {
+      document.dispatchEvent(new CustomEvent('exo-statistic-message', {
+        detail: {
+          module: 'Drive',
+          subModule: 'Documents',
+          userId: eXo.env.portal.userIdentityId,
+          userName: eXo.env.portal.userName,
+          name: 'actionCreateDescription',
+          operation: 'createDescription',
+          parameters: {
+            documentName: file.name,
+            category: this.file.folder ? 'Folder' : 'Document',
```

Comment:
```suggestion
            category: this.file.folder ? 'Folder' : 'Document',
```
You can i18n these values too by creating new keys in the properties file